backend/core/src/listings/listings.controller.ts

```diff
@@ -82,7 +87,15 @@ export class ListingsController {
   @Post()
   @ApiOperation({ summary: "Create listing", operationId: "create" })
   async create(@Body() listingDto: ListingCreateDto): Promise<ListingDto> {
-    return mapTo(ListingDto, await this.listingsService.create(listingDto))
+    const listing = await this.listingsService.create(listingDto)
+    /**
+     * clear list caches
+     * As we get more listings we'll want to update this to be more selective in clearing entries
+     */
+    for (const key of this.cacheKeys) {
```

Comment:
To clarify, this is clearing the entire cache whena new listing is created, right?

Reply:
Correct. Instead of clearing the cache we could update the cache entry to have the new listing.